pages/docs/cloud/repository-integration.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: "Grouparoo Cloud Repository Integration"
+date: "2021-11-10"
+pullQuote: "Simple integration steps for Grouparoo Cloud"
+---
+
+If you are here you should already have an API Key and Project ID.
+
+Grouparoo Cloud requires three things to function: A github action yaml file and two secrets to identify and authorize the code config repository.
+
+### Github Repository Secrets
+
+Grouparoo Cloud requires two secrets to be available in your repository: `GROUPAROO_CLOUD_TOKEN` and `GROUPAROO_CLOUD_PROJECT_ID`.
+
+If these values have already been provided for you, skip ahead. Otherwise you can get them from [Grouparoo Cloud](https://cloud.grouparoo.com/) in your Project's Integrations page.
+
+In your Github repository go to: Settings -> Secrets -> New repository secret.
```

Comment:
Let's show a screenshot here to make sure people know this is on the GitHub side. When trying to follow this with Andy, he got a bit confused on where exactly this was referring to